### PR TITLE
Set up a "Hello world" project as one app with Webpack

### PR DESCRIPTION
**Pull request by wandergithub**
- Created a new Rails app called 'hello-rails-react'.
      - Used the commend `rails new -j webpack my_new_rails_app` as described in the previous lesson's tutorial.
 - Verified that the `jsbundling-rails` gem was added to the Gemfile.
 - Initialized the project with Git.
 - Installed needed node dependencies as described in the previous lesson's tutorial.
 - Created a `babel.config.js` in the root folder of the project as described in the previous lesson's tutorial.
 - Updated `webpack.config.js` to include the babel-loader as described in the previous lesson's tutorial.
 - Set Postgres database. 
 - Created a `messages` table for storing messages and created 5 different greetings. 
 - Created an API endpoint that selects a random greeting from the  table.
 - Created a static view that is the root of the app.
- Created the `App` component with react-router and render it in the static view.
- Created the `Greeting` component that displays a greeting. Set it up as a route in the App component.
- Created a store, an action, and a reducer that connects to the API endpoint to get the random greeting.
- Display the random greeting in the `Greeting` component.

### CODE REVIEW

File: app/javascript/config.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { useDispatch } from "react-redux";
+
+// Actions
+const LOAD = "my-app/greetings/LOAD";
+// const CREATE = 'my-app/widgets/CREATE';
+// const UPDATE = 'my-app/widgets/UPDATE';
+// const REMOVE = 'my-app/widgets/REMOVE';
```
Comment (Nemwel-Boniface) [inline app/javascript/config.js]: - [x] Kindly remove all commented-out and un used/ unwanted pieces of code from your codebase. It makes it look messy and cluttered.

File: app/javascript/config.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+//   return { type: CREATE, widget };
+// }
+
+// export function updateWidget(widget) {
+//   return { type: UPDATE, widget };
+// }
+
+// export function removeWidget(widget) {
+//   return { type: REMOVE, widget };
+// }
```
Comment (Nemwel-Boniface) [inline app/javascript/config.js]: - [x] Kindly remove all commented-out and un used/ unwanted pieces of code from your codebase. It makes it look messy and cluttered.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello :wave: ,
I am Nemwel your reviewer for this PR :man_beard: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## :heavy_multiplication_x:  Invalid Code Review Request :heavy_multiplication_x: 

You have submitted a project, that does not use the new Microverse [README](https://github.com/microverseinc/readme-template) template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @wandergithub👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**